src/Collection.php

```diff
@@ -19,6 +19,8 @@
 use function iterator_to_array;
 use function sprintf;
 
+use const ReturnTypeWillChange;
```

Comment:
Thats what I meant in slack, you have to manually `use` the Attribute here (which is *not* a constant).

Codestyle will comply about the unused `ReturnTypeWillChange` `use`-Statement and will remove it when applying `cs-fix`.

To avoid that, you can add a class docblock:

```php
/**
 * @use ReturnTypeWillChange
 */
class Collection […]
```

Reply:
Fixed.